src/Sulu/Bundle/AdminBundle/Resources/js/components/Navigation/item.scss

```diff
@@ -31,6 +31,7 @@ $hoverColor: $shakespeare;
         background-color: transparent;
         text-align: left;
         font-weight: inherit;
+        cursor: pointer;
```

Comment:
The `item` already has cursor: pointer. Think the title does then not need it?

Reply:
The title is now an native button element and inherits styles from the user agent stylesheet. The default for button elements is `cursor: default`, which overwrites the inheritance from the cascade. Maybe the cursor styling can be removed from the item?

Reply:
Make sense as the `button` is the one with the click listener and not the `item`.